storage/src/main/java/org/apache/kafka/server/log/remote/storage/RemoteLogManagerConfig.java

```diff
@@ -134,6 +134,8 @@ public final class RemoteLogManagerConfig {
             "less than or equal to `log.retention.bytes` value.";
     public static final Long DEFAULT_LOG_LOCAL_RETENTION_BYTES = -2L;
 
+    public static final String REMOTE_LOG_METADATA_PREFIX = "remote.log.metadata";
```

Comment:
Should we append the `PROP` (or) `CONFIG` suffix to it?

Reply:
We already have `REMOTE_LOG_METADATA_MANAGER_CONFIG_PREFIX_PROP`. Configs that are prefixed with this value will be supplied to remote log metadata manager. 

To configure the values in the JIRA ticket:
```
remote.log.storage.manager.impl.prefix=remote.log.storage.
remote.log.metadata.manager.impl.prefix=rlmm.config.

rlmm.config.remote.log.metadata.topic.num.partitions=50
rlmm.config.remote.log.metadata.topic.replication.factor=4
rlmm.config.remote.log.metadata.topic.retention.ms=2592000000

remote.log.storage.s3...
```

Let's avoid one more `Map` (remoteLogMetadataProps) in RemoteLogManagerConfig. 

Also, we can define the default values for the config `prefix`  in this PR:
```
public static final String DEFAULT_REMOTE_STORAGE_MANAGER_CONFIG_PREFIX = "rsm.config.";
public static final String DEFAULT_REMOTE_LOG_METADATA_MANAGER_CONFIG_PREFIX = "rlmm.config.";
```

Reply:
Thanks for the good suggestion! PR updated.